### DISCUSSION TITLE
Website content not displaying on GitHub Pages

### DISCUSSION THREAD

**Discussion by Daniel-okwier** [community/community#78769] [New to GitHub] [answered]
Hi everyone,

I am experiencing an issue with my website deployed on GitHub Pages. The website is not displaying any content when accessed through the browser. I have followed the necessary steps to deploy my website, but I can't figure out what's causing this problem.

404
File not found

The site configured at this address does not contain the requested file.

If this is your site, make sure that the filename case matches the URL as well as any file permissions.
For root URLs (like http://example.com/) you must provide an index.html file.

[Read the full documentation](https://help.github.com/pages/) for more information about using GitHub Pages.

[GitHub Status](https://githubstatus.com/) — [@githubstatus](https://twitter.com/githubstatus)

my repository(https://github.com/Daniel-okwier/Netflix-project) 
my url(https://daniel-okwier.github.io/Netflix-project/)

**Comment by DeadGolden0** [comment, marked as answer]
Hello @Daniel-okwier ,

Since your repository contains a Netflix folder, your link is: 

> [!NOTE]
> https://daniel-okwier.github.io/Netflix-project/Netflix/index.html

You just need to add ``/Netflix`` after ``/Netflix-project``. 

If you want to remove this, simply delete your ``Netflix`` folder and upload your files directly to the repository.

Do not hesitate to mark the post as 'Answered' if my response suits you.

I hope my message is useful to you! Have a nice day!

**Reply by Daniel-okwier** [reply]
thank you

**Comment by 87diegoramirez** [comment]
https://www.tiktok.com/@87ramirezdiego?_t=8iBpTmT0hNu&_r=1
